config/7.10.0/common.yaml

```diff
@@ -159,6 +159,8 @@ console:
                     value: "[[.Auth.LDAP.BindCredential]]"
                   - name: AUTH_LDAP_JAAS_SECURITY_DOMAIN
                     value: "[[.Auth.LDAP.JAASSecurityDomain]]"
+                  - name: AUTH_LDAP_LOGIN_MODULE
+                    value: "[[.Auth.LDAP.LoginModule]]"
```

Comment:
what if nothing is set and the resulting `AUTH_LDAP_LOGIN_MODULE` var is blank? is this ok or should we be assigning a default value via code?

Reply:
the images set a default value if it is empty.